GooglePlayPlugins/com.google.play.billing/Samples/TrivialKart/scripts/Controller/Store/CarStorePageController.cs

```diff
@@ -0,0 +1,126 @@
+using System.Globalization;
+using System.Linq;
+using UnityEngine;
+using UnityEngine.UI;
+
+// controller for car store page
```

Comment:
you can do it later but make the class doc more descriptive:

Controls the car store page, do this, and this, and that